Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m

```diff
@@ -20,7 +20,9 @@ @interface ViewController () <MZDayPickerDelegate, MZDayPickerDataSource>
 @property (nonatomic, strong) NSString *searchText;
 @property (nonatomic, strong) NSArray *allEvents;
 @property (nonatomic, strong) NSDictionary *eventsDictionary;
```

Comment:
Two issues with this note:
1) There is no previous view controller
2) The reason that the allEvents array needs to be maintained is because it is the basis for creating the filteredEventsArray using the method arrayWithArray:[self.allEvents filteredArrayUsingPredicate:predicate]. I've checked, and there is no "filteredDictionaryUsingPredicate" method.